backend/layers/processing/publish_revisions.py

```diff
@@ -105,5 +105,7 @@ def run(self):
         S3Provider(),
         UriProvider(),
     )
-    do_not_publish_list = os.environ.get("DO_NOT_PUBLISH_LIST", None).split(",")
+    do_not_publish_list = os.environ.get("DO_NOT_PUBLISH_LIST", None)
```

Comment:
nit: your's works, just an alternative suggestion.
```suggestion
    do_not_publish_list = os.environ.get("DO_NOT_PUBLISH_LIST", "").split(",")
```

Reply:
This would result in a list of len 1 `[""]`. Which would work, but I just prefer having an explicit None when we don't set it